packages/squiggle-lang/src/fr/scale.ts

```diff
@@ -78,19 +87,33 @@ export const library = [
     output: "Scale",
     examples: [`Scale.symlog({ min: -10, max: 10 })`],
     definitions: [
-      makeDefinition([commonRecord], ([{ min, max, tickFormat }]) => {
-        checkMinMax(min, max);
+      makeDefinition(
+        [
+          frRecord(
+            ["min", frOptional(frNumber)],
+            ["max", frOptional(frNumber)],
+            ["tickFormat", frOptional(frString)],
+            ["constant", frOptional(frNumber)]
+          ),
+        ],
+        ([{ min, max, tickFormat, constant }]) => {
+          checkMinMax(min, max);
 
+          return vScale({
+            type: "symlog",
+            min: min ?? undefined,
+            max: max ?? undefined,
+            tickFormat: tickFormat ?? undefined,
+            constant: constant || SqSymlogScale.defaultConstant,
```

Comment:
It felt a bit weird to rely on SqSymlogScale, but it seemed more awkward (if even possible) to put this in ``vScale``

Reply:
Non-Sq* -> Sq* (`src/public/*`) dependency feels pretty wrong to me. We don't do this anywhere else (only `src/cli` depends on `src/public`, AFAICT).

I'd move it somewhere in `value/`, for example, you could `export class VScale` and then expose it as `static SYMLOG_DEFAULT_CONSTANT = ...`. Or export it as a plain constant in `value/index.ts`, or in `value/constants.ts`.

Or, if you think that `fr/scale` is more in control of this value and `value/index.ts` shouldn't be aware of it, then maybe export it from this file on top.

Reply:
> `constant || SqSymlogScale.defaultConstant`

What happens if `constant` is `0`? (It'd silently be replaced with `defaultConstant`, because you used `||` instead of `??`, I'm not sure if intentional)
What happens if `constant` is negative? Should be disallow both of those cases?